src/__tests__/internal/Pagination.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { shallow, mount, render } from 'enzyme';
+import { expect } from 'chai';
+import sinon from 'sinon';
+import Pagination from '../../components/internal/Pagination';
+
+    // Shallow Rendering
+    // https://github.com/airbnb/enzyme/blob/master/docs/api/shallow.md
+describe('Pagination - Shallow Rendering', () => {
+    it('to have the correct number of Pagination classes', () => {
+      const wrapper = shallow(
+        <Pagination
+          onChange={() => alert('Switch Page...')}
+          page={3}
+          pagesize={4}
+          numPages={6}
+        />,
+      );
+      expect(wrapper.find('.mars')).to.have.length(2);
+    });
+});
+
+  // Static Rendered Markup
+  // https://github.com/airbnb/enzyme/blob/master/docs/api/render.md
+describe('Pagination - Static Rendered Markup', () => {
+    it('renders the correct number of items?', () => {
+      const wrapper = render(
+        <Pagination
+          onChange={() => alert('Switch Page...')}
+          page={3}
+          pagesize={4}
+          numPages={6}
+        />,
+      );
+      expect(wrapper.find('.mars').length).to.equal(2);
```

Comment:
This should check that there are 4 items present on a page, at the moment you are checking that both pagination buttons (next/prev) are present

Reply:
Remove this test